### PR TITLE
Add addresses and link one to a face

### PR DESCRIPTION
**Pull request by kgehmlich**

### CODE REVIEW

File: OohInterview.DAL/Pocos/Face.cs [hunk 1]
```diff
@@ -7,10 +7,12 @@ public class Face
         public Guid Id { get; set; }
         public string Name { get; set; }
         public decimal RatePerDay { get; set; }
+        public Guid? AddressId { get; set; }
```
Comment (TomasMorton) [inline OohInterview.DAL/Pocos/Face.cs]: I like that it's optional 👍

File: OohInterview.DAL/Repositories/FaceRepository.cs [hunk 2]
```diff
@@ -22,4 +22,24 @@ public void AddFace(Face face)
             _dataContext.Faces.Add(face);
         }
     }
+
+    public class AddressRepository: IAddressRepository
```
Comment (TomasMorton) [inline OohInterview.DAL/Repositories/FaceRepository.cs]: Maybe should be in it's own file?

Comment (kgehmlich) [reply OohInterview.DAL/Repositories/FaceRepository.cs]: Woops, thought I moved that, thanks

**Review by TomasMorton — APPROVED** [review]

File: OohInterview.DAL/Pocos/Address.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace OohInterview.DAL.Pocos
+{
+    public class Address
+    {
+        public Guid Id { get; set; }
+        public string? Street { get; set; }
```
Comment (fokasu) [inline OohInterview.DAL/Pocos/Address.cs:8]: I would leave them the choice on what to put here...

**Pull request merged**